projects/wmf_wikivoyage.yaml

```diff
@@ -0,0 +1,201 @@
+paths:
+  /{api:v1}:
+    x-modules:
+        # swagger options, overriding the shared ones from the merged specs (?)
+      - spec:
+          info:
+            version: 1.0.0
+            title: Wikimedia REST API
+            description: >
+                This API provides cacheable and straightforward access to
+                Wikimedia content and data, in machine-readable formats.
+
+                ### Global Rules
+
+                - Limit your clients to no more than 200 requests/s to this API.
+                  Each API endpoint's documentation may detail more specific usage limits.
+                - Set a unique `User-Agent` or `Api-User-Agent` header that
+                  allows us to contact you quickly. Email addresses or URLs
+                  of contact pages work well.
+
+                By using this API, you agree to Wikimedia's
+                [Terms of Use](https://wikimediafoundation.org/wiki/Terms_of_Use) and
+                [Privacy Policy](https://wikimediafoundation.org/wiki/Privacy_policy).
+                Unless otherwise specified in the endpoint documentation
+                below, content accessed via this API is licensed under the
+                [CC-BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/)
+                and [GFDL](https://www.gnu.org/copyleft/fdl.html) licenses,
+                and you irrevocably agree to release modifications or
+                additions made through this API under these licenses.
+                See https://www.mediawiki.org/wiki/REST_API for background and details.
+
+                ### Endpoint documentation
+
+                Please consult each endpoint's documentation for details on:
+
+                - Licensing information for the specific type of content
+                  and data served via the endpoint.
+                - Stability markers to inform you about development status and
+                  change policy, according to
+                  [our API version policy](https://www.mediawiki.org/wiki/API_versioning).
+                - Endpoint specific usage limits.
+
+            termsOfService: https://wikimediafoundation.org/wiki/Terms_of_Use
+            contact:
+              name: the Wikimedia Services team
+              url: http://mediawiki.org/wiki/REST_API
+            license:
+              name: Apache2
+              url: http://www.apache.org/licenses/LICENSE-2.0
+          # Override the base path for host-based (proxied) requests. In our case,
+          # we proxy https://{domain}/api/rest_v1/ to the API.
+          x-host-basePath: /api/rest_v1
+          x-route-filters:
+            - path: lib/content_location_filter.js
+            - path: ./lib/normalize_title_filter.js
+              options:
+                redirect_cache_control: '{{options.purged_cache_control}}'
+          paths:
+            /page:
+              x-modules:
+                - path: v1/content.yaml
```

Comment:
We don't have pregenration enabled, right? So `no-store` is probably needed here as well?

Reply:
Idem